pkg/sentry/fs/proc/net.go

```diff
@@ -194,6 +195,199 @@ func (n *netDev) ReadSeqFileData(ctx context.Context, h seqfile.SeqHandle) ([]se
 	return data, 0
 }
 
+// netSnmp implements seqfile.SeqSource for /proc/net/snmp.
+//
+// +stateify savable
+type netSnmp struct {
+	s inet.Stack
+}
+
+// NeedsUpdate implements seqfile.SeqSource.NeedsUpdate.
+func (n *netSnmp) NeedsUpdate(generation int64) bool {
+	return true
+}
+
+type snmpLine struct {
+	prefix string
+	header string
+}
+
+var snmp = []snmpLine{
+	snmpLine{
```

Comment:
igudger: I don't think this is needed. Could you run `gofmt -w -s` on this file? The `-s` removes some unneeded code.
https://gvisor-review.git.corp.google.com/c/gvisor/+/18283/7/pkg/sentry/fs/proc/net.go#217

Reply:
Done.